app/index.js

```diff
@@ -12,6 +13,33 @@ import './app.global.css';
 import '../styles/main.less';
 import { ipcRenderer } from 'electron';
 import Raven from 'raven-js';
+import Rollbar from 'rollbar/dist/rollbar.umd';
+
+let rollbar = new Rollbar({
+    accessToken: '1843589282464f4facd43f794c8201a8',
```

Comment:
Should this be replaced with `ROLLBAR_POST_TOKEN`?

Reply:
This is actually the client-side token, which would end up distributed with JS code delivered to a browser environment and as such it's not as sensitive as the `ROLLBAR_POST_TOKEN`, so I didn't feel the need to go with ENV variables for it.

Reply:
OK, got it, just checking ;)